src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -77,15 +100,28 @@ function StripePaymentForm({
             },
           )
           .then(response => {
-            // TODO: This hits the receipt page twice, but Axios doesn't seem
-            // to support maxRedirects outside of NodeJS. Need to shift to a
-            // way that does not make two calls.
-            if (response.headers['content-type'].startsWith('text/html')) {
-              global.location.href = response.request.responseURL;
-            }
+            setLocation(response.data.receipt_page_url);
           })
           .catch(error => {
             console.log('[Project Zebra] POST error: ', error.message);
```

Comment:
We can remove this console.log now that you're using `setMessage`

Reply:
Done.